projects/1.16/assets/growthcraft-community-edition/growthcraft_cellar/lang/zh_cn.json

```
@@ -0,0 +1,105 @@
+   "fluid.growthcraft_cellar.copper_ale_fluid": "铜色啤酒",
+   "fluid.growthcraft_cellar.copper_lager_fluid": "铜色贮藏啤酒",
+   "fluid.growthcraft_cellar.dark_lager_fluid": "深色贮藏啤酒",
+   "fluid.growthcraft_cellar.ipa_ale_fluid": "印度淡色爱尔啤酒",
+   "fluid.growthcraft_cellar.old_port_ale_fluid": "老波特啤酒",
+   "fluid.growthcraft_cellar.pale_ale_fluid": "淡色爱尔啤酒",
+   "fluid.growthcraft_cellar.pale_lager_fluid": "淡色爱尔贮藏啤酒",
+   "fluid.growthcraft_cellar.pilsner_lager_fluid": "比尔森贮藏啤酒",
+   "fluid.growthcraft_cellar.stout_ale_fluid": "世涛啤酒",
+   "fluid.growthcraft_cellar.vienna_lager_fluid": "维也纳贮藏啤酒",
+   "fluid.growthcraft_cellar.wort_fluid": "麦芽汁",
+   "fluid.growthcraft_cellar.pale_golden_wort_fluid": "淡色爱尔金色麦芽汁",
+   "fluid.growthcraft_cellar.golden_wort_fluid": "金色麦芽汁",
+   "fluid.growthcraft_cellar.amber_wort_fluid": "琥珀色麦芽汁",
+   "fluid.growthcraft_cellar.deep_amber_wort_fluid": "深琥珀色麦芽汁",
+   "fluid.growthcraft_cellar.copper_wort_fluid": "铜色麦芽汁",
+   "fluid.growthcraft_cellar.deep_copper_wort_fluid": "深铜色麦芽汁",
+   "fluid.growthcraft_cellar.brown_wort_fluid": "棕色麦芽汁",
+   "fluid.growthcraft_cellar.dark_wort_fluid": "深色麦芽汁",
+   "fluid.growthcraft_cellar.red_grape_juice_fluid": "红葡萄果汁",
+   "fluid.growthcraft_cellar.white_grape_juice_fluid": "白葡萄果汁",
+   "fluid.growthcraft_cellar.purple_grape_juice_fluid": "紫葡萄果汁",
+   "fluid.growthcraft_cellar.red_wine_fluid": "红葡萄酒",
+   "fluid.growthcraft_cellar.purple_wine_fluid": "葡萄酒",
+   "fluid.growthcraft_cellar.white_wine_fluid": "白葡萄酒",
+   "fluid.growthcraft_cellar.hopped_golden_wort_fluid": "跃动金色麦芽汁",
+   "item.growthcraft_cellar.grape_red": "红葡萄",
+   "item.growthcraft_cellar.grape_purple": "紫葡萄",
+   "item.growthcraft_cellar.grape_white": "白葡萄",
+   "item.growthcraft_cellar.grape_seeds_purple": "紫葡萄种子",
+   "item.growthcraft_cellar.grape_seeds_red": "红葡萄种子",
+   "item.growthcraft_cellar.grape_seeds_white": "白葡萄种子",
+   "item.growthcraft_cellar.potion_ale": "一杯",
+   "item.growthcraft_cellar.potion_lager": "一罐",
+   "item.growthcraft_cellar.potion_wine": "一瓶",
+   "item.growthcraft_cellar.amber_ale_bucket": "琥珀色啤酒桶",
+   "item.growthcraft_cellar.amber_lager_bucket": "琥珀色贮藏啤酒桶",
+   "item.growthcraft_cellar.brown_ale_bucket": "棕色啤酒桶",
+   "item.growthcraft_cellar.brown_lager_bucket": "棕色贮藏啤酒桶",
+   "item.growthcraft_cellar.copper_ale_bucket": "铜色啤酒桶",
+   "item.growthcraft_cellar.copper_lager_bucket": "铜色贮藏啤酒桶",
+   "item.growthcraft_cellar.dark_lager_bucket": "深色贮藏啤酒桶",
+   "item.growthcraft_cellar.grain": "谷物",
+   "item.growthcraft_cellar.grain_golden": "金色烘烤谷物",
+   "item.growthcraft_cellar.grain_amber": "琥珀色烘烤谷物",
+   "item.growthcraft_cellar.grain_deep_amber": "深琥珀色烘烤谷物",
+   "item.growthcraft_cellar.grain_copper": "铜色烘烤谷物",
+   "item.growthcraft_cellar.grain_deep_copper": "深铜色烘烤谷物",
+   "item.growthcraft_cellar.grain_brown": "棕色烘烤谷物",
+   "item.growthcraft_cellar.grain_dark": "深色烘烤谷物",
+   "item.growthcraft_cellar.grain_pale_golden": "淡色爱尔金色烘烤谷物",

```

```suggestion
   "item.growthcraft_cellar.grain_pale_golden": "淡金色烘烤谷物",
```
这个是不是把`pale`统一替换成了`淡色爱尔`（）